docs/conf.py

```diff
@@ -311,3 +311,5 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
+
+autodoc_member_order = 'bysource'
```

Comment:
I'd like some feedback on this change. I'm hoping it might make things a bit more logical to sort through... not sure.

Reply:
Both `bysource` (more logical order) and `alphabetical` (easier to find things) make sense to me. Too bad users can not change this dynamically.